### PR TITLE
{chem}[intel/2016a] ADF v2014.11.r48287

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

depends on https://github.com/hpcugent/easybuild-easyblocks/pull/826

### CODE REVIEW

**Comment by boegel** [comment]
@wpoely86: please review?

File: easybuild/easyconfigs/a/ADF/ADF-2014.11.r48287-intel-2016a.eb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+name = 'ADF'
+version = '2014.11.r48287'
+
+homepage = 'http://www.scm.com/ADF/'
+description = """ADF is an accurate, parallelized, powerful computational chemistry program to understand and
+ predict chemical structure and reactivity with density functional theory (DFT)."""
+
+toolchain = {'name': 'intel', 'version': '2016a'}
+
```
Comment (wpoely86) [inline easybuild/easyconfigs/a/ADF/ADF-2014.11.r48287-intel-2016a.eb]: add a dummy license file like for icc in comment?

Comment (boegel) [inline easybuild/easyconfigs/a/ADF/ADF-2014.11.r48287-intel-2016a.eb]: done below

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/0ede54c4aecd93c275e8 for a full test report.

**Comment by boegel** [comment]
Jenkins: test this please

**Comment by boegel** [comment]
Going in, thanks for the review @wpoely86!

**Pull request merged**